cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -274,7 +274,18 @@
   (is (= (reformat-string "(foo bar) \n(foo baz)" {:remove-trailing-whitespace? false})
          "(foo bar) \n(foo baz)"))
   (is (= (reformat-string "(foo bar) \n" {:remove-trailing-whitespace? false})
-         "(foo bar) \n")))
+         "(foo bar) \n"))
+  (is (= (reformat-string "( \nfoo\n )"
+                          {:remove-surrounding-whitespace? false
+                           :remove-trailing-whitespace? false})
+         "( \n foo\n )"))
+  (is (= (reformat-string "(foo\n \n)"
```

Comment:
This is PEZ use case. I renamed his "bar" example to "foo" for consistency with other tests.